Source/SwiftLintFramework/Rules/CyclomaticComplexityRule.swift

```diff
@@ -20,13 +21,16 @@ public struct CyclomaticComplexityRule: ASTRule, ConfigProviderRule {
         description: "Complexity of function bodies should be limited.",
         nonTriggeringExamples: [
             "func f1() {\nif true {\nfor _ in 1..5 { } }\nif false { }\n}",
-            "func f3() {while true {}}",
+            "func f(code: Int) -> Int {" +
+            "switch code {\n case 0: fallthrough\ncase 0: return 1\ncase 0: return 1\n" +
```

Comment:
these should be indented to indicate that they're additions to the string above.